### DISCUSSION TITLE
What is the use for "Location type" in stock locations?

### DISCUSSION THREAD

**Discussion by lifeisafractal** [inventree/InvenTree#7612] [Q&A] [answered]
I haven't been able to dig up the use for stock location's "location type". I can't seem to find any reference to it in the docs or in the UI beyond creating types and assigning them.

Without knowing what they are I was hoping for something like the following. I have inventory for multiple clients and need to keep it separated and be able to allocate build orders from specific client's stock. Right now I use sublocations with a top level location for each client. I was hoping location type might help with this. I'd like to keep the tree structure of the locations to purely represent where things are physically stored, and not use it to shoehorn in ownership of stock. I was hoping I could do something like select a specific location type to allocate from when doing auto allocation of builds.

Any way, just curious what that feature is intended for.

**Comment by wolflu05** [comment, marked as answer]
I added the storage location type feature in the first place to inherit the icon. E.g. I have an InvenTree location for every drawer on a shelf, box, .... Now I have one location type for each size of drawer, box or shelf.  That way the icons of the hundredths of same sized drawers are in sync.

Later I have in mind to add size information to each location type to add some functionality to quickly find empty drawers in a location which would be capable to store something with specific dimensions that are given to the helper.

**Reply by lifeisafractal** [reply]
Thanks for the info. Sounds like what you implemented it for doesn't really match my use case. Just a suggestion, I don't think this is mentioned anywhere in the docs and updating them to call this out could be helpful.

**Reply by wolflu05** [reply]
Agreed, the documentation definitely needs to mention those location types. I added a small note in #7652 now.

**Reply by lifeisafractal** [reply]
Awesome, thanks for getting that in!
